epoch.go

```diff
@@ -1282,7 +1282,9 @@ func (e *Epoch) voteOnBlock(block Block) (Vote, error) {
 }
 
 func (e *Epoch) increaseRound() {
-	e.Logger.Info(fmt.Sprintf("Moving to a new round (%d --> %d", e.round, e.round+1), zap.Uint64("round", e.round+1))
+	leader := LeaderForRound(e.nodes, e.round)
+	e.Logger.Info(fmt.Sprintf("Moving to a new round (%d --> %d", e.round, e.round+1),
```

Comment:
fwiw - I think it's probably an anti-pattern to include formatted strings into a structured log output (as the log won't be able to be easily matched by a consumer)

Reply:
ok I removed the formatting